config/config.go

```diff
@@ -28,7 +28,7 @@ type Config struct {
 	CloudStoryEnabled bool   `json:"cloud_story_enabled"`
 	ExistingMap       string `json:"existing_map"`
 	EnableMapping     bool   `json:"enable_mapping"`
-	RunSlam           bool   `json:"run_slam"`
+	RunSlam           *bool  `json:"run_slam"`
```

Comment:
This needs to be a pointer so that we can distinguish between it being not in the config and being int he config but set to false. Otherwise, by default, all existing slam configs will no longer boot cartographer.

Reply:
shouldn't we just use the feature flag? then once the project is done we can migrate configs